docs/components/connectors/out-of-the-box-connectors/sql.md

```diff
@@ -31,6 +31,7 @@ Select the database type you want to connect to. The **SQL connector** supports
 - Microsoft SQL Server
 - MySQL
 - PostgreSQL
+- **Oracle:** The Oracle Database Connector requires the Oracle JDBC driver, which Camunda cannot distribute due to licensing restrictions. To connect to an Oracle database, you must manually download the JDBC driver from [Oracle](https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html) and run the connector in [hybrid mode](/components/connectors/use-connectors-in-hybrid-mode.md). When building a custom Docker image, include the driver by copying it into the image—for example, add `COPY ojdbc17.jar /opt/custom/` to your Dockerfile. This ensures the driver is on the classpath when the connector runtime starts.
```

Comment:
Great addition. What about using an [admotion](https://docusaurus.io/docs/markdown-features/admonitions) for this part?

We can include `Oracle` in the list and add the admotion for it below.

Reply:
WDYT about the suggestion I've made?

Reply:
Look good, thank you :)